src/blob.c

```diff
@@ -79,7 +79,7 @@ static int write_file_stream(
 	ssize_t read_len = -1, written = 0;
```

Comment:
Need to make "written" a git_off_t rather than ssize_t so that the while loop will sum correctly.